### PR TITLE
HCCDOC-2426 Edit utterances

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: data/domain.yml [hunk 1]
```diff
@@ -189,14 +189,14 @@ responses:
   utter_services_offline_no_incidents:
     - text: "All services seem to be operating normally."
   utter_services_offline_error:
-    - text: "I was unable to talk with [status.redhat.com](https://status.redhat.com) to fulfill your request."
+    - text: "I was unable to communicate with [status.redhat.com](https://status.redhat.com) to fulfill your request."
```
Comment (emmurphy1) [inline data/domain.yml:192]: Just a suggestion. Maybe you have a good reason for using 'talk'?

Comment (justinorringer) [reply data/domain.yml:192]: Only varying the diction a little bit. It's hard not to use the same words over and over.

File: data/domain.yml [hunk 2]
```diff
@@ -123,11 +123,11 @@ responses:
           payload: /intent_insights_advisor_recommendation_by_type
   utter_core_rephrase:
     - text: "I'm sorry. I didn't understand that. Could you try to rephrase your question?"
-    - text: "Sorry I didn't get that. Can you rephrase?"
-    - text: 'I don''t understand. Say "I need help" for information on what I can do.'
+    - text: "Sorry, I didn't get that. Can you rephrase?"
+    - text: 'I don't understand. Say "I need help" for information on what I can do.'
```
Comment (justinorringer) [inline data/domain.yml:127]:
```suggestion
    - text: 'I don''t understand. Say "I need help" for information on what I can do.'
```

That extra `'` is required in this case, needs to escape the other one.

File: data/domain.yml [hunk 3]
```diff
@@ -83,17 +83,17 @@ actions:
 
 responses:
   utter_pii_warning:
-    - text: "*You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool. Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is intended to assist with general queries. Please see [Red Hat Terms of use](https://www.redhat.com/en/about/terms-use) and [Privacy Statement](https://www.redhat.com/en/about/privacy-policy).*"
+    - text: "*You are about to use the Red Hat Hybrid Cloud Console virtual assistant chat tool. Please do not include any personal or confidential information in your interaction with the virtual assistant. The tool is intended to assist with general queries. Please see [Red Hat Terms of use](https://www.redhat.com/en/about/terms-use) and [Privacy Statement](https://www.redhat.com/en/about/privacy-policy).*"
```
Comment (josejulio) [inline data/domain.yml:86]: This was given by legal - I'll revert the changes in this utterance.
```suggestion
    - text: "*You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool. Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is intended to assist with general queries. Please see [Red Hat Terms of use](https://www.redhat.com/en/about/terms-use) and [Privacy Statement](https://www.redhat.com/en/about/privacy-policy).*"
```

**Review by josejulio — APPROVED** [review]
lgtm, just a little thing that we won't change now 

Thanks!!

File: data/domain.yml (hunk 1 above)
Comment (emmurphy1) [reply data/domain.yml:192]: Sure, but it's easier for users if we are consistent with language. I know it's also boring :-)
